walk-in-interview/src/main/webapp/new-job/index.html

```diff
@@ -66,5 +69,7 @@
 </body>
 
 <script type="module" src="./script.js"></script>
+<!-- TODO(issue/69): find a way to hide the api key -->
```

Comment:
https://developers.google.com/maps/api-key-best-practices

FYI: typically the key should not be checked in. it should be a configuration variable, probably served by the server and injected dynamically in your scripts.

Reply:
okay sure, for now I've just added restrictions on the websites that can use this api key